packages/web/app/src/pages/target-laboratory.tsx

```diff
@@ -17,7 +18,7 @@ import {
   AccordionContent,
   AccordionHeader,
   AccordionItem,
-  AccordionTrigger,
+  AccordionTriggerPrimitive,
```

Comment:
@dimaMachina do you think there's a better approach to it?
I exported there and imported here a primitive component, to avoid overwriting className a lot.

Reply:
I think it's fine like that